_layouts/default.html

```diff
@@ -8,6 +8,9 @@
   <main id="main-content" class="margin-top-0 margin-bottom-0">
     {% include notice.html %}
     {{ content }}
+    {% if page.page-helpful != false %}
```

Comment:
```suggestion
    {% if page.page-helpful %}
```

Reply:
The poll won't show up if I do this because the attribute is only set if it's false.